src/FixedDataTableRow.js

```diff
@@ -153,6 +152,17 @@ class FixedDataTableRowImpl extends React.Component {
     touchEnabled: PropTypes.bool,
   };
 
+  shouldComponentUpdate(/*object*/ nextProps) /*boolean*/ {
+    // when scrolling stops, nextProps.isScrolling will be false, so we use this.props.isScrolling to avoid unnecessary rendering
+    // where row index is same during vertical scrolling
+    return (
+        (!nextProps.isScrolling && !this.props.isScrolling) ||
```

Comment:
Same as the above comment about the scrolling part.
Also can we use 
```js
!(nextProps.isScrolling || this.props.isScrolling)
```

Btw, will this skip other updates to the row props, such as width, rowHeight, columnReorderingData, etc. while scrolling? Not sure if they could lead to problems.

Reply:
> It looks like it might skip those props even if we're not scrolling...

If we take over the approach in [FixedDataTableCellGroup](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/src/FixedDataTableCellGroup.js#L195), skipping of prop changes will be done while scrolling. This might have a better performance than checking ALL the props, with the cost of loosing updates, but only while scrolling. 

We could also keep the scrolling check, and if it fails we then check for prop changes. This might be a good trade off between both (we'll still loose updates while scrolling though...) This is the approach at FixedDataTableCell.js when pureRendering is turned on.

> We should perhaps instead update this to be a pure component / check across all props.

If this is done, then maybe it can also be done to FixedDataTableCellGroup.js ?

Reply:
I will probably make separate PR for `shouldComponentUpdate` changes as suggested by @wcjordan